bin/generate-tiles

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+set -o nounset
+
+# For backward compatibility, allow both PG* and POSTGRES_* forms,
+# with the non-standard POSTGRES_* form taking precedence.
+# An error will be raised if neither form is given, except for the PGPORT
+export PGHOST="${POSTGRES_HOST:-${PGHOST?}}"
+export PGDATABASE="${POSTGRES_DB:-${PGDATABASE?}}"
+export PGUSER="${POSTGRES_USER:-${PGUSER?}}"
+export PGPASSWORD="${POSTGRES_PASSWORD:-${PGPASSWORD?}}"
+export PGPORT="${POSTGRES_PORT:-${PGPORT:-5432}}"
+
+# List of postgres servers
+if [[ -z "${PGHOSTS_LIST}" ]] # "xxx.xxx.xxx.xxx&host=xxx.xxx.xxx.xxx&host=..."
+then
+  export HOST_COUNT=1
+  export PGHOSTS="${PGHOST}"
+else
+  export HOST_COUNT=`awk -F"&" '{print NF}' <<< "${PGHOSTS_LIST}"`
+  export PGHOSTS="${PGHOSTS_LIST}"
+fi
+
+export NO_GZIP=${NO_GZIP:-1}
+if [[ ${NO_GZIP} ]] && [[ ! ${NO_GZIP} == "0" ]] && [[ ! ${NO_GZIP} == "true" ]]
+then
+  export NO_GZIP=1
+fi
+
+export KEY=${KEY:-1}
+if [[ ${KEY} ]] && [[ ! ${KEY} == "0" ]] && [[ ! ${KEY} == "true" ]]
+then
+  export KEY=1
+fi
+
+export FUNCZXY="${FUNCZXY:-getmvt}"
+export COPY_CONCURRENCY=${COPY_CONCURRENCY:-1}  # number of CPUs per postgres server
+export MAXCONNECTIONS=${MAXCONNECTIONS:-${COPY_CONCURRENCY}}
```

Comment:
We should clarify MAX_HOST_CONNECTIONS is per host.  Sorry all the nitpicking, but naming is hard -- once we merge, we won't be able to easily change those :)

```suggestion
export MAX_HOST_CONNECTIONS=${MAX_HOST_CONNECTIONS:-${COPY_CONCURRENCY}}
```

Reply:
Renamed.